### DISCUSSION TITLE
Please add MPC-BE as a video wallpaper player option

### DISCUSSION THREAD

**Discussion by vermi5** [rocksdanister/lively#2492] [Ideas]
Or any other FOSS DirectShow player that can use MPC Video Renderer because, as of right now, there seems to be no other way to get RTX Video HDR to tonemap SDR videos with nvidia >3000 gpus

**Comment by rocksdanister** [comment]
Actually had DirectShow support during the early builds but discontinued due to the setup being additional step for regular users.

**Reply by vermi5** [reply]
Gotcha, I haven't found a way for Lively to interface with MPC-BE besides CLI options anyway, so controlling playback speed and such could be problematic.
Besides, as it turns out, CefSharp correctly triggers RTX Video HDR when playing Youtube URLs so I'm big chillin.

Thanks boss.
